src/tinypeel/tinypeel.py

```diff
@@ -28,8 +28,7 @@ def runPeelingCycles(pedigree, peelingInfo, args, singleLocusMode=False):
         #     print("Estimating the transmission rate is currently a disabled option")
         # PeelingUpdates.updateSeg(peelingInfo) #Option currently disabled.
 
-        if args.esterrors:
-            PeelingUpdates.updatePenetrance(pedigree, peelingInfo)
+        PeelingUpdates.updatePenetrance(pedigree, peelingInfo, args)
```

Comment:
Please add the condition `if args.est_geno_error_prob or args.est_seq_error_prob:` back